docs/devguide/flashfirmware.rst

```diff
@@ -33,15 +32,17 @@ some convenience targets.
 Preparing firmware and a Python program
 ---------------------------------------
 
-tools/makecombined
-
-hexlify
-
+Try these::
 
+  tools/makecombined
+  hexlify
```

Comment:
I think with makecombined.py you don't need hexlify.py, so the latter doesn't need to be mentioned.  You could actually mention in more detail how to use makecombined, eg: ./tools/makecombined.py build/bbc-microbit-classic-gcc-nosd/source/microbit-micropython.hex myscript.py